docker/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+version: '3'
```

Comment:
My docker only seems to understand up to version '2' of the config format.
Replacing '3' with '2' doesn't seem to result in any obvious problems for me.
Can we use version '2' here? Or do you know that you are using version '3' features of docker?